Adding animations 11 and 12 (#3354)
GenericMappingTools/gmt: doc/rst/source/animations/anim12.rst

.. _anim12:

> (10) Image and Grid Manipulations

Reviewer: ```suggestion
(12) Image and Grid Manipulations
```

=================================

> We again us grdmix, blending the NASA day and night views from the Blue and Black Marble mosaic

Reviewer: ```suggestion
We again use :doc:`/grdmix`, blending the NASA day and night views from the Blue and Black Marble mosaic
```

images, but here we recompute a day-night mask for every 1 minute in a single day, for 1440
frames.  In addition, we adjust the colors using the intensities derived

> from the slopes of the earth relief grid.  Since no GMT plot is made here we do not use :doc:`movie`

Reviewer: ```suggestion
from the slopes of the earth relief grid.  Since no GMT plot is made here we do not use :doc:`/movie`
```

> but instead use :doc:`batch to build the images and then create the movie directly from the set

Reviewer: ```suggestion
but instead use :doc:`/batch` to build the images and then create the movie directly from the set
```
